tasks/src/image-segmentation/data.ts

```diff
@@ -6,8 +6,7 @@ import { TASKS_MODEL_LIBRARIES } from "../const";
 const taskData: TaskData = {
 	datasets: [
 		{
-			// TODO write proper description
-			description: "Benchmark dataset used for the task",
```

Comment:
We're keeping the dataset and model descriptions really short because there's not much space for them in the page, you could still change but please keep this in mind to keep it short :)

Reply:
WDYT of 

> Widely used benchmark dataset for multiple Vision tasks